Task2/style2.css

```
@@ -0,0 +1,251 @@
+  body {
+    padding: 0;
+    margin: 0;
+    font-family: 'Times New Roman', Georgia, serif;
+  }
+  
+  h1, h2, h3 {  
+    letter-spacing: 5px;
+    font-weight: normal;
+  }
+  
+  .container {
+    max-width: 1600px;
+    min-width: 500px;
+    margin: auto;
+    position: fixed;
+    width: 100%;
+    z-index: 1;
+    background-color: white;
+    box-shadow: 0px 0px 30px #8a8585;
+    padding: 8px 16px;
+  }
+       
+  .brand {
+    font-weight: bold;
+    font-size: xx-small;
+    letter-spacing: 3px;
+    float: left;
+    display: block;
+    font-family: "Playfair Display";
+    opacity: 0.75;
+  } 
+  
+  .nav {
+    float: right;
+  }
+  
+  .nav ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: white;
+  }
+  
+  .nav li {
+    float: left;
+  }
+  
+  .nav li a {
+    display: block;
+    color: black;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+  }
+  
+  .nav li a:hover:not(.active) {
+    background-color: #ddd;
+  }
+  
+  .nav li a.active {
+    color: white;
+    background-color: #4CAF50;
+  }
+  
+  #imagetext {
+    font-size: xx-large;
+    display: bottom;
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    opacity: 0.60;
+    letter-spacing: 4px;
+    padding-left: 45px; 
+    font-weight: lighter;
+  }
+  
+  .firstsec img {
+    max-width: 100%;
+    height: auto;
+  }
+  
+  .content {
+    max-width: 1100px;
+    display: block;
+    box-sizing: inherit;
+    padding: 64px;
+    margin: 0 auto;
+  }
+  
+  .secondsec {
+    margin-top: 30px;
+    margin-bottom: 30px;
+    padding-top: 64px;
+    padding-bottom: 64px;
+    box-sizing: border-box;
+    display: flex;
+  }
+  
+  .secondsec .left {
+    vertical-align: top;
+    flex: 50%;
+    padding: 15px;
+    width: 100%;
+    opacity: 0.75;
+  }
+  
+  .img.main image {
+    padding-top: 60px;
+    padding-bottom: 20px;
+    width: 600px;
+    height: 750px;
+  }
+  
+  .secondsec .right {
+    display: inline-block;
+    flex: 50%; 
+    padding: 15px;
+    width: 100%;
+    font-size: 18px;
+    line-height: 1.5;
+  }
+  
+  .secondsec .right h2,
+  .secondsec .right h3 {
+    text-align: center;
+    letter-spacing: 3px;
+    font-family: "Playfair Display";
```

Remove unwanted font family.
```suggestion
    font-family: "Playfair Display";
```